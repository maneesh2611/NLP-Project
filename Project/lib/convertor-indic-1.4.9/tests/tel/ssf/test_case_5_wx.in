<Sentence id="1">
1	hExarAbAx	NNP	<fs af='hExarAbAx,n,,sg,,d,0,0'>|<fs af='hExarAbAx,n,,sg,,o,ti,ti'>
2	lo	PSP	<fs af='lo,avy,,,,,,'>|<fs af='la,avy,,,,,o,o_avy'>
3	guru	NN	<fs af='guruvu,n,,sg,,,I,I'>|<fs af='guruvu,n,,sg,,o,ti,ti'>
4	,	SYM	<fs af=',,punc,,,,,'>
5	SukravArAllo	NN	<fs af='SukravAraM,n,,pl,,,lo,lo'>
6	jastis	NN	<fs af='jastis,n,,sg,,d,0,0'>|<fs af='jastis,n,,sg,,o,ti,ti'>
7	SrIkqRNa	NNP	<fs af='SrIkqRNa,n,,sg,,d,0,0'>|<fs af='SrIkqRNa,n,,sg,,o,ti,ti'>|<fs af='SrIkqRNudu,n,,sg,,o,ti,ti'>
8	kamitI	NN	<fs af='kamitI,n,,sg,,d,0,0'>|<fs af='kamitI,n,,sg,,d,0,0'>|<fs af='kamitI,n,,sg,,o,ti,ti'>|<fs af='kamitI,n,,sg,,o,ti,ti'>
9	paryatana	NN	<fs af='paryatana,n,,sg,,d,0,0'>|<fs af='paryatana,n,,sg,,o,ti,ti'>|<fs af='paryatanaM,n,,sg,,o,ti,ti'>
10	kevalaM	NN	<fs af='kevalaM,n,,sg,,d,0,0'>|<fs af='keva,n,,pl,,d,nu,nu'>
11	paricayAniki	VM	<fs af='paricayaM,n,,sg,,,ki,ki'>
12	,	SYM	<fs af=',,punc,,,,,'>
13	avagAhana	NN	<fs af='avagAhana,n,,sg,,d,0,0'>|<fs af='avagAhana,n,,sg,,o,ti,ti'>|<fs af='avagAhanaM,n,,sg,,o,ti,ti'>
14	kosaM	PSP	<fs af='kosaM,avy,,,,,0,0_avy'>
15	mAwrame	PSP	<fs af='mAwrame,avy,,,,,,'>|<fs af='mAwraM,avy,,,,,e,e_avy'>
16	ani	UT	<fs af='ani,avy,,,,,0,0_avy'>|<fs af='anu,v,any,any,any,,i,i'>
17	kamitI	NN	<fs af='kamitI,n,,sg,,d,0,0'>|<fs af='kamitI,n,,sg,,d,0,0'>|<fs af='kamitI,n,,sg,,o,ti,ti'>|<fs af='kamitI,n,,sg,,o,ti,ti'>
18	saBya	NN	<fs af='saByaM,n,,sg,,o,ti,ti'>|<fs af='saByudu,n,,sg,,o,ti,ti'>
19	kAryaxarSi	NN	<fs af='kAryaxarSi,n,,sg,,d,0,0'>|<fs af='kAryaxarSi,n,,sg,,o,ti,ti'>
20	vinox	VM	<fs af='vinox,unk,,,,,,'>
21	keV	NN	<fs af='keV,avy,,,,,0,0_avy'>
22	xuggal	NN	<fs af='xuggal,unk,,,,,,'>
23	weVlipAru	VM	<fs af='weVlupu,v,fm,pl,3,,A,A'>
24	.	SYM	<fs af='.,punc,,,,,,'>
</Sentence>
