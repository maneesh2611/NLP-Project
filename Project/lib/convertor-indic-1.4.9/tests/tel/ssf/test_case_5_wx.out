<Sentence id="1">			
1	హైదరాబాద్	NNP	<fs af='హైదరాబాద్,n,,sg,,d,0,0'>|<fs af='హైదరాబాద్,n,,sg,,o,టి,ti'>
2	లో	PSP	<fs af='లో,avy,,,,,,'>|<fs af='ల,avy,,,,,ఓ,o_avy'>
3	గురు	NN	<fs af='గురువు,n,,sg,,,ఈ,I'>|<fs af='గురువు,n,,sg,,o,టి,ti'>
4	,	SYM	<fs af=',,punc,,,,,'>
5	శుక్రవారాల్లో	NN	<fs af='శుక్రవారం,n,,pl,,,లో,lo'>
6	జస్టిస్	NN	<fs af='జస్టిస్,n,,sg,,d,0,0'>|<fs af='జస్టిస్,n,,sg,,o,టి,ti'>
7	శ్రీకృష్ణ	NNP	<fs af='శ్రీకృష్ణ,n,,sg,,d,0,0'>|<fs af='శ్రీకృష్ణ,n,,sg,,o,టి,ti'>|<fs af='శ్రీకృష్ణుడు,n,,sg,,o,టి,ti'>
8	కమిటీ	NN	<fs af='కమిటీ,n,,sg,,d,0,0'>|<fs af='కమిటీ,n,,sg,,d,0,0'>|<fs af='కమిటీ,n,,sg,,o,టి,ti'>|<fs af='కమిటీ,n,,sg,,o,టి,ti'>
9	పర్యటన	NN	<fs af='పర్యటన,n,,sg,,d,0,0'>|<fs af='పర్యటన,n,,sg,,o,టి,ti'>|<fs af='పర్యటనం,n,,sg,,o,టి,ti'>
10	కేవలం	NN	<fs af='కేవలం,n,,sg,,d,0,0'>|<fs af='కేవ,n,,pl,,d,ను,nu'>
11	పరిచయానికి	VM	<fs af='పరిచయం,n,,sg,,,కి,ki'>
12	,	SYM	<fs af=',,punc,,,,,'>
13	అవగాహన	NN	<fs af='అవగాహన,n,,sg,,d,0,0'>|<fs af='అవగాహన,n,,sg,,o,టి,ti'>|<fs af='అవగాహనం,n,,sg,,o,టి,ti'>
14	కోసం	PSP	<fs af='కోసం,avy,,,,,0,0_avy'>
15	మాత్రమే	PSP	<fs af='మాత్రమే,avy,,,,,,'>|<fs af='మాత్రం,avy,,,,,ఏ,e_avy'>
16	అని	UT	<fs af='అని,avy,,,,,0,0_avy'>|<fs af='అను,v,any,any,any,,ఇ,i'>
17	కమిటీ	NN	<fs af='కమిటీ,n,,sg,,d,0,0'>|<fs af='కమిటీ,n,,sg,,d,0,0'>|<fs af='కమిటీ,n,,sg,,o,టి,ti'>|<fs af='కమిటీ,n,,sg,,o,టి,ti'>
18	సభ్య	NN	<fs af='సభ్యం,n,,sg,,o,టి,ti'>|<fs af='సభ్యుడు,n,,sg,,o,టి,ti'>
19	కార్యదర్శి	NN	<fs af='కార్యదర్శి,n,,sg,,d,0,0'>|<fs af='కార్యదర్శి,n,,sg,,o,టి,ti'>
20	వినోద్	VM	<fs af='వినోద్,unk,,,,,,'>
21	కె	NN	<fs af='కె,avy,,,,,0,0_avy'>
22	దుగ్గల్	NN	<fs af='దుగ్గల్,unk,,,,,,'>
23	తెలిపారు	VM	<fs af='తెలుపు,v,fm,pl,3,,ఆ,A'>
24	।	SYM	<fs af='।,punc,,,,,,'>
</Sentence>			
